tests/PoolModel.js

```diff
@@ -62,6 +62,23 @@ describe('Pool Model Tests', function () {
         });
     });
 
+    it('should generate a list of pool start/end dates for 2018', () => {
```

Comment:
This is a unit test for the PoolFactory itself, rather than the PoolModel. We should extract this to a separate file and/or set up a PoolModel test that verifies it can create Pool instances correctly from the PoolFactory.